libs/client/dashboard/src/lib/components/tags-management/tag-form/tag-form.component.ts

```diff
@@ -67,8 +67,8 @@ export class TagFormComponent implements OnInit {
         const parentVal = (this.fields.parent.value === this.NO_PARENT)? null : this.fields.parent.value;
 
         const form: TagsForm = {
-            name: this.fields.name.value,
-            desc: this.fields.desc.value,
+            name: (this.fields.name.value as string).trim(),
+            desc: (this.fields.desc.value as string).trim(),
```

Comment:
minor heebie-jeebies here--if `this.fields.desc.value` can be null, this will crash. I _think_ we're saved by Angular enforcing that the fields of a form default to an empty string instead of null, but not 100% sure that that's always the case.